### ISSUE TITLE
Building on Windows with Python27

### ISSUE THREAD

**Issue by rcmpayne** [iOSForensics/pymobiledevice#16]
any ideas?

```
C:\CODE\pymobiledevice-master>C:\Python27\python setup.py build
running build
running build_py
file pymobiledevice.py (for module pymobiledevice) not found
file pymobiledevice.py (for module pymobiledevice) not found
```

**Comment by GotoHack** [comment]
Could you try with the latest version ?

**Comment by GotoHack** [comment]
Should be fixed with commit 879a728.
